android/build.gradle

```diff
@@ -11,8 +11,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-  compileSdkVersion 23
-  buildToolsVersion "23.0.1"
+  compileSdkVersion 25
+  buildToolsVersion "25.0.1"
```

Comment:
@Minishlink: hmm, what would be the downside of using `compileSdkVersion` as `25`?